src/components/NewPost/NewPost.js

```diff
@@ -19,6 +20,9 @@ const NewPost = (props) => {
 
   const submitHandler = (event) => {
     event.preventDefault();
+    axios.post('/posts.json', inputState)
+      .then(response => console.log(response))
+      .catch(error => console.log(error));
     props.onPostAdded(inputState);
```

Comment:
A jeśli jednak pojawi się error? 
.onPostAdded() i zerowanie inputa powinno wykonywać się jednak po części then(). Np. po sprawdzeniu kodu zwróconego w response. 
Przy okazji pytanie: jaki kod powinien przyjść w response w odpowiedzi na POST?  :)

Reply:
Może zrobię modal, który wyświetli się wraz z numerem błędu? Czy narazie wstawić poprostu alert? Czy coś innego?

Rzeczywiście, przeniosę te funkcje :)

Jeśli tworzyłabym nowe źródło bazy danych, to powinien to być kod 201. Tutaj baza jest już stworzona, więc dodając kolejny element ją uaktualniam, więc jest kod 200. Dobrze to rozumiem?

Reply:
Ogólnie na chwilę obecną logowanie błędu w konsoli mogłoby wystarczyć ;) Mi raczej chodziło o to, że w przypadku powyższego kodu: w konsoli pojawi się error, ale post w samej aplikacji zostanie dodany. Do najbliższego odświeżenia, bo skoro nie dodał się do bazy, zniknie. 
W rzeczywistości mógłby się pojawiać jakiś alert z informacją, że wystąpił błąd. Np. nad formularzem. Żeby ktoś mógł ponowić próbę zapisu formularza. 
Ale mi tak naprawdę chodziło tylko o to pierwsze - by użytkownik nie był wprowadzany w błąd, że zdjęcie dodało się do bazy, gdy się nie dodało ;) 

Kody: nie do końca :) To nie chodzi o tworzenie nowego źródła danych, a tworzenie nowego "rekordu" w bazie. POST może zwrócić jedną z kilku odpowiedzi: 200, 201, 202 lub 204.  Firebase decyduje, że w odpowiedzi jest 200 OK i informacja zwrotna: https://firebase.google.com/docs/reference/rest/database#section-post . Ale mogłoby być tam też 201, że rekord został stworzony. Porównaj sobie lekcje 167 oraz 181 z kursu Maxa :) 
Tylko zapamiętaj, że to nie chodzi o tworzenie bazy, a tworzenie pojedynczych rekordów i odpowiedź na tą akcję :)

We wtorek pokażę Ci fajne narzędzie do testowania komunikacji z bazą danych ;)

Reply:
Teraz rozumiem, o co chodziło z tymi błędami :) zaraz podeślę zmiany :)